docker/README.md

```diff
@@ -65,6 +76,78 @@ $ docker-compose pull
 $ docker-compose up
 ```
 
+## Use of .env file
+The docker-compose file has variables for the image tags. This allows the use of a ``.env`` file.
+By default, ``branch-main`` will be used. The following is an example of a file overriding some services :
```

Comment:
```suggestion
By default, ``branch-main`` will be used. The following is an example of a file overriding some services:
```

It'd be better to default to `latest` as that's, well, the default behaviour.

Reply:
done